conftest.py

```diff
@@ -21,3 +24,10 @@ def set_fuser(fuser):
 
 def pytest_configure(config):
     set_fuser(config.getoption("fuser"))
+
+def pytest_benchmark_update_machine_info(config, machine_info):
+    machine_info['pytorch_version'] = torch.__version__
+    machine_info['torchtext_version'] = torchtext.__version__
+    machine_info['torchvision_version'] = torchvision.__version__
```

Comment:
Why not trying to import those only inside the function and throw an just fill the version as not installed if import fails?
```suggestion
    try:
      import torchtext
      machine_info['torchtext_version'] = torchtext.__version__
    catch ImportError:
      machine_info['torchtext_version'] = '*not-installed*'
     try:
      import torchvision
      machine_info['torchvision_version'] = torchvision.__version__
    catch ImportError:
      machine_info['torchvision_version'] = '*not-installed*'    
```